consensus/src/phase.rs

```diff
@@ -47,21 +47,22 @@ pub enum Phase<T: Operations, D: Database> {
 
 impl<T: Operations + 'static, D: Database + 'static> Phase<T, D> {
     pub fn initialize(&mut self, msg: &Message, round: u64, step: u8) {
-        trace!(
-            "init phase:{} with msg {:?} at round:{} step:{}",
+        debug!(
+            "event: init_step:{} round:{} step:{} msg: {:#?}",
```

Comment:
Should you change this to be formatted the same as the rest?
I mean with `event = "init_step", round, step, msg = format!("{msg:#?}")`

Reply:
Indeed, this is covered in next issue.